common/types/offer.go

```diff
@@ -141,6 +148,25 @@ func (o *Offer) validate() error {
 		return errExchangeRateNil
 	}
 
+	// We want to prevent offers whose claim value is so low that a relayer
+	// can't be used to complete the swap if the maker does not have sufficient
+	// ETH to make the claim themselves. While relayers are not used with ERC20
+	// swaps, we still want a minimum swap amount, so we use the same value.
+	relayerFeeAsXMR, err := o.ExchangeRate.ToXMR(coins.RelayerFeeETH)
+	if err != nil {
+		return err
+	}
+	if o.MinAmount.Cmp(relayerFeeAsXMR) <= 0 {
+		return fmt.Errorf(
+			"min amount must be greater than %s ETH when converted (%s XMR)",
+			coins.RelayerFeeETH.Text('f'), relayerFeeAsXMR.Text('f'))
```

Comment:
maybe suggest the minimum xmr amount allowed in the error message?

Reply:
We print both values. The first value in ETH is a constant.  I treated the XMR value as secondary, putting it in parentheses, because it is calculated and depends on the exchange rate in the offer. If you have a suggestion on how to improve the wording, let me know.